hack/pr-deploy-backporter.yaml

```diff
@@ -0,0 +1,203 @@
+  apiVersion: image.openshift.io/v1
+  metadata:
+    namespace: ci-tools-${PULL_REQUEST}
+    name: output
+  spec:
+    lookupPolicy:
+      local: true
+- apiVersion: build.openshift.io/v1
+  kind: BuildConfig
+  metadata:
+    name: bp-server
+    namespace: ci-tools-${PULL_REQUEST}
+  spec:
+    output:
+      to:
+        kind: ImageStreamTag
+        name: output:bp-server
+    source:
+      contextDir: images/bugzilla-backporter
+      git:
+        ref: ${BRANCH}
+        uri: https://github.com/${USER}/ci-tools.git
+      images:
+      - from:
+          kind: ImageStreamTag
+          namespace: ci-tools-${PULL_REQUEST}
+          name: "output:bp-binaries"
+        paths:
+        - destinationDir: images/bugzilla-backporter/.
+          sourcePath: /go/bin/bugzilla-backporter
+      type: Git
+    strategy:
+      dockerStrategy:
+        imageOptimizationPolicy: SkipLayers
+      type: Docker
+    triggers:
+    - type: ImageChange
+      imageChange:
+        from:
+          kind: ImageStreamTag
+          namespace: ci-tools-${PULL_REQUEST}
+          name: "output:bp-binaries"
+- apiVersion: build.openshift.io/v1
+  kind: BuildConfig
+  metadata:
+    name: bp-binaries
+    namespace: ci-tools-${PULL_REQUEST}
+  spec:
+    output:
+      to:
+        kind: ImageStreamTag
+        name: "output:bp-binaries"
+    source:
+      dockerfile: |
+        FROM registry.svc.ci.openshift.org/openshift/release:golang-1.13
+
+        COPY . .
+        RUN go install ./cmd/bugzilla-backporter/...
+      git:
+        ref: ${BRANCH}
+        uri: https://github.com/${USER}/ci-tools.git
+      contextDir: ./
+      type: Git
+    strategy:
+      dockerStrategy:
+        imageOptimizationPolicy: SkipLayers
+      type: Docker
+    triggers:
+    - type: ConfigChange
+- kind: ImageStream
+  apiVersion: image.openshift.io/v1
+  metadata:
+    name: bugzilla-backporter
+    namespace: ci-tools-${PULL_REQUEST}
+  spec:
+    lookupPolicy:
+      local: true
+    tags:
+    - name: latest
+      importPolicy:
+        scheduled: true
+      from:
+        kind: DockerImage
+        name: registry.svc.ci.openshift.org/ci/bugzilla-backporter
+- apiVersion: v1
+  kind: Service
+  metadata:
+    namespace: ci-tools-${PULL_REQUEST}
+    labels:
+      app: bp-server
+    name: bp-server
+  spec:
+    ports:
+    - name: main
+      port: 80
+      protocol: TCP
+      targetPort: main
+    selector:
+      app: bp-server
+- apiVersion: route.openshift.io/v1
+  kind: Route
+  metadata:
+    labels:
+      app: bp-server
+    name: bp-server
+    namespace: ci-tools-${PULL_REQUEST}
+  spec:
+    tls:
+      termination: edge
+      insecureEdgeTerminationPolicy: Redirect
+    port:
+      targetPort: main
+    to:
+      kind: Service
+      name: bp-server
+- apiVersion: v1
+  kind: Secret
+  metadata:
+    name: bugzilla-credentials-openshift-bugzilla-robot
+    namespace: ci-tools-${PULL_REQUEST}
+  type: Opaque 
+  data:
+    api: o
```

Comment:
I need an empty file at "/etc/bugzilla/api". Will having a data element named "api" create that file there - since I am using the secret at the "/etc/bugzilla" volume?

Reply:
Will the server even work without a valid token? You could instead add to the Makefile the `oc` commands necessary to copy the credential from the `ci` namespace into the `ci-tools-${PULL_REQUEST}` one, as we do for config maps.

Reply:
The server should show the view even without the keys - its only for the create clones(ie. write) functionality that the token is needed